src/models/imposter.js

```diff
@@ -54,6 +54,9 @@ function create (Protocol, request) {
 
                 result.requests = server.requests;
                 result.stubs = server.stubs();
+                result.stubs.forEach(function (stub) {
```

Comment:
I could not find a way to inject the imposter state into response handlers and predicates functions without having reference in the stub to that state. If I could get a reference from the stub to it's imposter it would solve this less than clean code.
Suggestions?